code/modules/roguetown/roguecrafting/alchemy/cauldron_recipes.dm

```diff
@@ -127,3 +127,10 @@
 	smells_like = "calming"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/buff/fortune = 27)
+
+//Ultra SPECIAL potion - Rotcure. Removes rot-penalty and revive sickness.
+/datum/alch_cauldron_recipe/curerot_potion
```

Comment:
none of the ingredients can actually be put into the cauldron, I'm not even sure what the point of this recipe is, since the one above is a table recipe and you haven't added any alchemical potion elements for this potion